nextflow.config

```diff
@@ -35,7 +35,7 @@ params {
     amp_skip_macrel                 = false
 
     amp_skip_ampir                  = false
-    amp_ampir_model                 = 'precursor'
+    amp_ampir_model                 = 'mature'
```

Comment:
Just out of curiosity: Why is the 'mature' model better as default?

Reply:
In the `meta` mode, its more likely that you would detect the `mature` peptides, at least in our case. We already have `meta` invoked in the prodigal/prokka module n the pipeline, so i thought that would make more sense in this case.

Reply:
After looking at the documentation we think actualy that `precusor` may still be better as a default value. We will revert this but document in a summary fashion which to use when.